les/client_handler.go

```diff
@@ -114,11 +114,25 @@ func (h *clientHandler) handle(p *serverPeer) error {
 		p.Log().Debug("Light Ethereum handshake failed", "err", err)
 		return err
 	}
+	// Register peer with the server pool
+	if h.backend.serverPool != nil {
+		if nvt, err := h.backend.serverPool.RegisterNode(p.Node()); err == nil {
+			p.setValueTracker(nvt)
```

Comment:
Will you consider that we can move the "valueTracker" notion into the `vflux/client` totally?

IIUC, we just need to feed some statistics for the `valueTracker` and all the magic can be hidden in the `vflux/client`. The benefit is we can totally get rid of the `nodeValueTracker *vfc.NodeValueTracker` in the peer.

Reply:
So we can simplify the logic like 

```go
// Register peer with the server pool
	if err := h.backend.serverPool.RegisterNode(p.Node()); err != nil {
		return err
	}
	defer h.backend.serverPool.UnregisterNode(p.Node())
```